packages/components/src/molecules/Dropdown/DropdownButton.tsx

```diff
@@ -1,51 +1,64 @@
-import React, { forwardRef, useImperativeHandle, AriaAttributes } from 'react'
-import Button, { ButtonProps } from '../../atoms/Button'
-
-import { useDropdown } from './hooks/useDropdown'
+import React, { cloneElement, forwardRef, ReactNode } from 'react'
+import Button, { ButtonProps, IconPosition } from '../../atoms/Button'
+import { useDropdownTrigger } from './hooks/useDropdownTrigger'
 
 export interface DropdownButtonProps
-  extends Omit<ButtonProps, 'variant' | 'inverse'> {
+  extends Omit<ButtonProps, 'variant' | 'inverse' | 'icon' | 'iconPosition' > {
   /**
    * ID to find this component in testing tools (e.g.: cypress, testing library, and jest).
    */
   testId?: string
   /**
-   * For accessibility purposes, add an ARIA label to the element when it doesn't have a label.
+   * Replace the default rendered element with the provided child element, merging their props and behavior.
+   */
+  asChild?: boolean
+  /**
+   * Boolean that represents a loading state.
+   */
+  loading?: boolean
+  /**
+   * Specifies a label for loading state.
+   */
+  loadingLabel?: string
+  /**
+   * @deprecated
+   * A React component that will be rendered as an icon.
+   */
+  icon?: ReactNode
+  /**
+   * @deprecated
+   * Specifies where the icon should be positioned
    */
-  'aria-label'?: AriaAttributes['aria-label']
```

Comment:
question: was it handled somewhere else?

Reply:
Not that I saw, but it still works as expected. The ButtonProps already includes the aria-label prop, so it seems to be redundant."